### PR TITLE
docs(@vtmn/svelte): badge component

### PR DESCRIPTION
**Pull request by Tlahey**
Add some documentation how to install and use the svelte badge

### CODE REVIEW

**Review by lauthieb — APPROVED** [review]

File: packages/showcases/svelte/stories/components/indicators/VtmnBadge/VtmnBadge.stories.svelte [hunk 1]
```diff
@@ -13,57 +11,17 @@
...
+  {...parameters}
 />
 
 <Template let:args>
   <VtmnBadge {...args} />
 </Template>
 
 <Story
-  name="Overview"
+  name="Playground"
```
Comment (lauthieb) [inline packages/showcases/svelte/stories/components/indicators/VtmnBadge/VtmnBadge.stories.svelte:22]:
```suggestion
  name="Overview"
```

Comment (Tlahey) [reply packages/showcases/svelte/stories/components/indicators/VtmnBadge/VtmnBadge.stories.svelte:22]: Yes my bad

**Review by lauthieb — APPROVED** [review]

File: packages/showcases/svelte/stories/components/indicators/VtmnBadge/documentation.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+## Value
+
+There are 3 different types of display
+
+- When value is undefined or 0
+- When value is between 1 and 99
+- When value is upper than 99
+
+<Canvas>
+  <Story name="No value">
```
Comment (lauthieb) [inline packages/showcases/svelte/stories/components/indicators/VtmnBadge/documentation.stories.mdx:40]: Why it is named "No value" ?

**Review by lauthieb — CHANGES_REQUESTED** [review]

File: packages/showcases/svelte/stories/components/indicators/VtmnBadge/documentation.stories.mdx (hunk 2 above)
Comment (Tlahey) [reply packages/showcases/svelte/stories/components/indicators/VtmnBadge/documentation.stories.mdx:40]: Humm I can rename to "Equal to zero"

Comment (Tlahey) [reply packages/showcases/svelte/stories/components/indicators/VtmnBadge/documentation.stories.mdx:40]: With our discussion, the conversation are resolved ?

File: packages/showcases/svelte/stories/components/indicators/VtmnBadge/documentation.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  </Story>
+  <Story name="More than 99">
+    {{
+      Component: VtmnBadge,
+      props: { value: 100 },
+    }}
+  </Story>
+</Canvas>
+
+# Variants
```
Comment (lauthieb) [inline packages/showcases/svelte/stories/components/indicators/VtmnBadge/documentation.stories.mdx:60]:
```suggestion
## Variants
```

**Review by lauthieb — APPROVED** [review]

**Pull request merged**